docs/concepts/how-ipfs-works.md

```diff
@@ -68,7 +75,7 @@ While [the subsystems described above](#how-ipfs-represents-data) handle the rep
 
 ### Kademlia Distributed Hash Table (DHT)
 
-IPFS uses Kademlia, a <VueCustomTooltip label="A decentralized data store that maps data based on key-value pairs." underlined multiline is-left>Distributed Hash Table (DHT)</VueCustomTooltip> designed for decentralized peer-to-peer computer networks. Kademlia is used to map what the user is looking for to the peer that is storing the matching content. The Kademlia DHT can be thought as a huge table that stores information on who has what data, and where that data might be located. The Kademlia DHT provides IPFS with:
+IPFS uses Kademlia, a <VueCustomTooltip label="A decentralized data store that maps data based on key-value pairs." underlined multiline is-left>Distributed Hash Table (DHT)</VueCustomTooltip> designed for decentralized peer-to-peer computer networks. Kademlia is used to map what the user is looking for to the peer that is storing the matching content. The Kademlia DHT can be thought of as a large table distributed across many nodes that stores information on who has what data, and where that data might be located. The Kademlia DHT provides IPFS with:
 
 - A catalog and a navigation system for data on the IPFS network
```

Comment:
The main thing that the Kademlia DHT provides is a way to store key/val records without a central server. The brilliance is how the XOR "distance" metric (where distance is peer ID XOR record key )is used to create a self-organising system with pretty efficient lookups (log n).


https://www.youtube.com/watch?v=3Y6vLTzM7zA
https://kelseyc18.github.io/kademlia_vis/basics/1/